src/profile.c

```diff
@@ -11,10 +11,11 @@ void printReadIt(RedisModule_Reply *reply, IndexIterator *root, size_t counter,
   IndexReader *ir = root->ctx;
 
   RedisModule_Reply_Map(reply);
-
   if (ir->idx->flags == Index_DocIdsOnly) {
-    printProfileType("TAG");
-    REPLY_KVSTR_SAFE("Term", ir->record->term.term->str);
+    if (ir->record->term.term != NULL) {
+      printProfileType("TAG");
+      REPLY_KVSTR_SAFE("Term", ir->record->term.term->str);
+    }
```

Comment:
Is this also a bug?

Reply:
No, this was added since the new inverted index uses the `Index_DocIdsOnly` flag as well as the `TAG` index, which was the only one to use it till now.
I will add some documentation. Maybe I'll find some clearer condition.